core/scripts/others/helper.py

```diff
@@ -15,6 +15,15 @@ def linux_path(path):
     return path
 
 
+def get_wcosa_path():
+    """returns the absolute path of wcosa"""
+    return linux_path(os.path.abspath("./../../"))
```

Comment:
This gives different values depending on where the scripts are being called from.

Reply:
I know the fill will patch it in a bit